chore: v3.2 Release Notes (#1620)
umee-network/umee: RELEASE_NOTES.md

Highlights:

- IBC update to v5.1

> - `QueryLiquidationTargets` RPC is now available when the node is run with `--enable-liquidator-query`. The LIQUIDATOR build flag has been removed. NOTE: the future should not be enabled for nodes with public API. The query involves intensive computation and can impact node stability when used by an attacker.

Reviewer: ```suggestion
- `QueryLiquidationTargets` RPC is now available when the node is run with `--enable-liquidator-query`. The LIQUIDATOR build flag has been removed. NOTE: this query should not be enabled for nodes with public API. The query involves intensive computation and can impact node stability when used by an attacker.
```

> - Introduced experimental features, available when build with `experimental` flag. It must not be used with a mainnet.

Reviewer: ```suggestion
- Introduced experimental features, available when build with `experimental` flag. This flag must not be used on mainnet.
```
